applications/MeshingApplication/tests/mmg_eulerian_test/coarse_sphere_test_result.sol

```diff
@@ -0,0 +1,97 @@
+MeshVersionFormatted 2
```

Comment:
do you need the .mesh file in the repository?

Reply:
I removed the mesh file, that's one of the things

Reply:
I replaced by a sol file, smaller